framework/encode/capture_manager.cpp

```diff
@@ -456,6 +456,13 @@ void CaptureManager::EndApiCallCapture()
             WriteToFile(parameter_buffer->GetHeaderData(),
                         parameter_buffer->GetHeaderDataSize() + parameter_buffer->GetDataSize());
         }
+
+        // Flush after presents to avoid capture files with incomplete final
+        // blocks.
+        if (thread_data->call_id_ == format::ApiCall_vkQueuePresentKHR)
+        {
+            file_stream_->Flush();
```

Comment:
What do you think about moving this to `CaptureManager::EndFrame` so the flush is run for any Vulkan or DX12 API call that GFXR handles as a frame delimiter?

Reply:
It sounds like a good idea. I have a vague feeling that there was a reason that I didn't do that but I cannot remember what it was. I'll try the change and let CI chew on it. Thanks!